Update README.md

rui-2016 opened the pull request:
Ad two papers on frequency capping in branch "Budget Pacing & Frequency/Recency Capping";
Ad a new branch "market segmentation" in "Demand-Side Platform (DSP) Techniques", and add two papers on market segmentation in the branch.

rui-2016 committed:
Update README.md

Ad two papers on frequency capping in branch "Budget Pacing & Frequency/Recency Capping";
Ad a new branch "market segmentation" in "Demand-Side Platform (DSP) Techniques", and add two papers on market segmentation in the branch.

wnzhang commented:
Thanks for updating.
Two points:
1. It looks the author name 'Qi' is wrong.
2. The papers are on ieee website, which is not accessible for everyone. Could you please uploaded them onto an open URL?

rui-2016 committed:
Update README.md

Ad two papers on frequency capping in branch "Budget Pacing & Frequency/Recency Capping";
Ad a new branch "market segmentation" in "Demand-Side Platform (DSP) Techniques", and add three papers on market segmentation in the branch;
Ad one paper on inventory allocation in branch "Supply-Side Platform (SSP) Techniques".
